react/rules/AUT.js

```diff
@@ -0,0 +1,70 @@
+import { isPastDate } from '../utils/dateRules'
+
+export default {
+  country: 'AUT',
+  personalFields: [
+    {
+      name: 'firstName',
+      maxLength: 100,
+      label: 'firstName',
+      required: true,
+    },
+    {
+      name: 'lastName',
+      maxLength: 100,
+      label: 'lastName',
+      required: true,
+    },
+    {
+      name: 'email',
+      maxLength: 100,
+      label: 'email',
+      hidden: true,
+    },
+    {
+      name: 'document',
+      maxLength: 50,
+      label: 'document',
+      required: false,
+    },
+    {
+      name: 'homePhone',
+      maxLength: 30,
+      label: 'homePhone',
+    },
+    {
+      name: 'gender',
+      maxLength: 30,
+      label: 'gender',
+    },
+    {
+      name: 'birthDate',
+      maxLength: 30,
+      label: 'birthDate',
+      type: 'date',
+      validate: isPastDate
```

Comment:
There is no specific mask for the Australian date format?

Reply:
Hi Gabriel, the only request was to add the country rules for Austria, but no extra details were requested, so I used what most countries use.

Reply:
We tend not to add details not requested directly because sometimes the country may accept different formats (as it looks like to be the case of [Austria](https://en.wikipedia.org/wiki/List_of_date_formats_by_country)) and different clients may have different preferences, so they may customize it as needed.